### PR TITLE
Update TermdbTest with values for survival terms

### PR DESCRIPTION
**Pull request by creilly8**
## Description
Closes issue #1991. Corresponding PR -> https://github.com/stjude/sjpp/pull/459. 

Changes: 
1. `values` added to both `Event-free survival` and `Overall survival` in TermdbTest
2. Matrix integration tests updated to reflect change. Matrix now shows values instead of exit code for TermdbTest. 

Test: 
1. [issue #1991 violin plot test link](http://localhost:3000/?noheader=1&mass=%7B%22dslabel%22:%22TermdbTest%22,%22genome%22:%22hg38-test%22,%22nav%22:%7B%22activeTab%22:1%7D,%22plots%22:%5B%7B%22chartType%22:%22summary%22,%22term%22:%7B%22id%22:%22agedx%22,%22q%22:%7B%22mode%22:%22continuous%22%7D%7D,%22term2%22:%7B%22id%22:%22efs%22%7D%7D%5D%7D)
2. [TermdbTest matrix](http://localhost:3000/?noheader=1&mass={%22dslabel%22:%22TermdbTest%22,%22genome%22:%22hg38-test%22,%22nav%22:{%22header_mode%22:%22hidden%22},%22plots%22:[{%22chartType%22:%22matrix%22,%22termgroups%22:[{%22lst%22:[{%22term%22:{%22gene%22:%22TP53%22,%22type%22:%22geneVariant%22}},{%22term%22:{%22gene%22:%22AKT1%22,%22type%22:%22geneVariant%22}},{%22id%22:%22efs%22,%22q%22:{%22mode%22:%22continuous%22}},{%22id%22:%22agedx%22,%22q%22:{%22mode%22:%22continuous%22}},{%22id%22:%22diaggrp%22},{%22term%22:{%22gene%22:%22TP53%22,%22type%22:%22geneExpression%22}}]}],%22divideBy%22:{%22id%22:%22sex%22}}]})
3. [Passed manual integration test run](https://github.com/stjude/proteinpaint/actions/runs/10709268113)

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

**Review by karishma-gangwani — APPROVED** [review]

File: client/mass/test/matrix.integration.spec.js [hunk 1]
```diff
@@ -2817,7 +2817,7 @@ tape('apply "hide" and "show" legend filters to a survival term', function (test
 
 		// 1. Hide
 		const legendTexts = [...matrix.Inner.dom.legendG.node().querySelectorAll('g g text')].find(d =>
-			d?.__data__?.text?.startsWith('Exit code: 0')
+			d?.__data__?.text?.startsWith('No Event')
```
Comment (gavrielm) [inline client/mass/test/matrix.integration.spec.js:2820]: May need to be changed now that labels in terms table have been changed

**Review by congyu-lu — COMMENTED** [review]
from the termdbMatrix link, edit a survival term will show an error (getting density plot)

**Comment by creilly8** [comment]
> from the termdbMatrix link, edit a survival term will show an error (getting density plot)

@congyu-lu, I'm seeing the same problem in master. Would you please confirm if you see the same?

**Comment by congyu-lu** [comment]
> > from the termdbMatrix link, edit a survival term will show an error (getting density plot)
> 
> @congyu-lu, I'm seeing the same problem in master. Would you please confirm if you see the same?

Yes I also see the error in master

**Review by congyu-lu — APPROVED** [review]
survival term edit density plot error is not related to this PR

**Review by gavrielm — APPROVED** [review]

**Pull request merged**